eng/tools/typespec-validation/package.json

```diff
@@ -20,7 +21,7 @@
     },
     "scripts": {
         "build": "tsc",
-        "postinstall": "npm run build",
+        "postinstall": "cd ../suppressions && npm run build && cd ../typespec-validation && npm run build",
```

Comment:
This is a little ugly, but the best we can do for now.  If you have an NPM dependency on an internal "self-building" package, you also need to build it before youself.  Only direct deps, not transitive.